build/build-from-repos.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# this script assumes that in
+# ~/bromite  ---> there is the bromite repo
+# ~/chromium ---> there is the chromium repo
+
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+cd ~
+
+echo -e ${RED} -------- download bromite repo ${NC}
+mkdir ~/bromite
+cd ~/bromite
+git fetch
+git pull
+
+VERSION=$( cat ~/bromite/build/RELEASE )
+echo -e ${RED} -------- lastest version is: $VERSION ${NC}
+
+
+echo -e ${RED} -------- download chromium repo ${NC}
+mkdir ~/chromium
+cd ~/chromium
+fetch --nohooks --no-history android
+
+
+cd src
+
+# cleanup repo!
+git reset
+git reset --hard HEAD
+git clean -fdx
+git fetch --depth 2 https://chromium.googlesource.com/chromium/src.git +refs/tags/$VERSION:chromium_$VERSION
+git tag
+git checkout .
+git clean -fdx
+git checkout $VERSION
+
+echo -e ${RED} -------- git current status ${NC}
+git status
+git log
+
+echo -e ${RED} -------- sync other chromium repos ${NC}
+gclient sync -D
+
+# remove origin for chromium
+git remote remove origin
+
+echo -e ${RED} ------- apply patchs ${NC}
+for file in $(cat ~/bromite/build/patches_list.txt) ; do 
+	echo -e ${RED} " -> Apply $file" ${NC}
+
+	REPL="0,/^---/s//FILE:"$file"\n---/"
```

Comment:
@csagan5 
i tried with tags and notes before, but sadly git-rebase doesn't keep those values.
so I preferred to add a description in the commit, in the form

`FILE: <patch name>`

which is then deleted with export-all-patch.sh (https://github.com/bromite/bromite/pull/687/files#diff-0a85d7dc8b850b3e4ac402a503566661R17)

Reply:
Yeah it looks like solid work; I only released once some basic scripts: https://github.com/bromite/bromite/blob/for-fdroid/fdroid/build.sh

They are very outdated by now; then others picked up from there and did their own scripts (see other comment). I also have some scripts but I do not release them because then I would have to maintain them. Building Chromium is cumbersome but once you can build that then adding the patches should be easy.